proteinnpt/baselines/model.py

```diff
@@ -66,7 +66,7 @@ def __init__(self, args, alphabet):
         if self.args.target_prediction_head == "AA_embeddings_mean_pooled":
             target_pred_input_dim = self.args.embed_dim
         elif self.args.target_prediction_head == "One_hot_encoding":
-            target_pred_input_dim = (self.args.seq_len + 1) * self.alphabet_size if args.target_prediction_model!="CNN" else self.alphabet_size    #Add one for the BOS token
+            target_pred_input_dim = (self.args.seq_len + 2) * self.alphabet_size if args.target_prediction_model!="CNN" else self.alphabet_size    #Add one for the BOS token
```

Comment:
Why extending by 2 tokens here?

Reply:
Default alphabet is `ESM-1b` (`train.py` line 210). This sets both `prepend_bos` and `append_eos` to True when instantiating `AugmentedPropertyPredictor`, so tokenized sequences are `seq_len + 2` long.

Reply:
Got it - I must have been using the MSA Transformer Alphabet before refactoring the code for open release!